nmostesting/suites/BCP0060102Test.py

```diff
@@ -21,98 +25,883 @@ class BCP0060102Test(ControllerTest):
+                                                                     ('min_bit_rate', 2983000),
+                                                                     ('max_bit_rate', 7955000),
+                                                                     ('level', '8k-2'),
+                                                                     ('conformance_level', 'UHD2'),
+                                                                     ('interop_point', '2a')])
+        interoperability_points.append(interop_point_2a)
+
+        interop_point_2b = self._create_interop_point(ip1a_2c_base, [('video_width', 7680), ('video_height', 4320),
+                                                                     ('bit_rate', 4000000),
+                                                                     ('min_bit_rate', 2983000),
+                                                                     ('max_bit_rate', 7955000),
+                                                                     ('video_transfer_characteristic', 'PQ'),
+                                                                     ('level', '8k-2'),
+                                                                     ('conformance_level', 'UHD2'),
+                                                                     ('interop_point', '2b')])
+        interoperability_points.append(interop_point_2b)
+
+        interop_point_2c = self._create_interop_point(ip1a_2c_base, [('video_width', 7680), ('video_height', 4320),
+                                                                     ('bit_rate', 4000000),
+                                                                     ('min_bit_rate', 2983000),
+                                                                     ('max_bit_rate', 7955000),
+                                                                     ('video_transfer_characteristic', 'HLG'),
+                                                                     ('level', '8k-2'),
+                                                                     ('conformance_level', 'UHD2'),
+                                                                     ('interop_point', '2c')])
+        interoperability_points.append(interop_point_2c)
+
+        return interoperability_points
+
+    def _generate_non_JXSV_interop_points(self, size):
+        # Non JPEG XS Interoperability Points
+        non_jxsv_base = {'media_type': 'raw', 'capability_set': None, 'conformance_level': None}
+
+        interoperability_points = []
+
+        for i in range(0, size):
+            interop_point = deepcopy(non_jxsv_base)
+            interop_point['interop_point'] = 'NonJXSV' + str(i)
+            interoperability_points.append(interop_point)
+
+        return interoperability_points
+
+    def _generate_constraint_set(self, sdp_params):
+        split_exactframerate = sdp_params.get("video_exactframerate",
```

Comment:
Apply same dict unpacking trick to default all params to SDP_PREFERENCES in one go?